apps/backend/src/resolvers/mutations/AssignMembersToFapMutation.ts

```diff
@@ -125,6 +131,14 @@ export class AssignMembersToFapMutation {
     );
   }
 
+  @Mutation(() => Fap)
+  async massAssignReviews(
```

Comment:
Maybe it is better to use `Fap` keyword in the mutation name. Something like `massAssignFapReviews` sounds better for me because it is much more clear when you use it from the frontend, or in the schema playground, that this is Fap related reviews and not some other review.